docs/examples/settings.html

```diff
@@ -0,0 +1,166 @@
+	<meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
+
+	<link type="text/css" rel="stylesheet"
+		href="../dist/css/bootstrap.min.css" />
+	<link type="text/css" rel="stylesheet" href="scratch.css" />
+
+	<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+	<script src="../dist/js/bootstrap.min.js"></script>
+
+	<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js"></script>
+
+	<script src="forms.js"></script>
+
+	<script>
+	angular.module('ExampleApp', ['tw.styleguide-components']);
+	</script>
+
+	<script src="js/nav.directive.js"></script>
+	<script src="js/simple-nav.directive.js"></script>
+	<script src="js/side-nav.directive.js"></script>
+	<script src="js/start-new.directive.js"></script>
+	<script src="js/activity-filters.directive.js"></script>
+	<script src="js/footer.directive.js"></script>
+
+
+	<script src="//transferwise.github.io/styleguide-components/dist/js/styleguide-components.js"></script>
+	<link type="text/css" rel="stylesheet" href="//transferwise.github.io/styleguide-components/components/currency-flags/dist/currency-flags.min.css" />
+
+	<script>
+	(function(angular) {
+
+		angular.module('ExampleApp')
+		.controller('PageController', [
+			PageController
+		]);
+
+		function PageController() {
+			var $ctrl = this;
+
+			$ctrl.burgerToggle = function() {
+				$ctrl.open = !$ctrl.open;
+			};
+			$ctrl.showSearch = false;
+
+			$ctrl.sideNavToggle = function() {
+				$ctrl.sideNav = !$ctrl.sideNav;
+			};
+			$ctrl.sideNav = true;
+		}
+	})(window.angular);
+	</script>
+	<style>
+	.nav-tabs {
+		border-bottom: 0;
+		white-space: nowrap;
+		overflow-x: auto;
+		overflow-y: hidden;
+	}
+
+	.nav-tabs > .active > a {
+		border-bottom: 0;
+	}
+	.nav-tabs .active {
+		border: 0;
+		background-color: #FFF;
+		border-top-left-radius: 3px;
+		border-top-right-radius: 3px;
+	}
+	.nav-tabs > li {
+		float: none;
+		display: inline-block;
+	}
+	.nav-tabs > li > a {
+		padding: 12px 32px;
+	}
+	.tab-pane {
+		background-color: #FFF;
+		padding: 32px;
+	}
+	.tab-content {
+		margin-top: -2px;
+	}
+	</style>
+</head>
+<body ng-app="ExampleApp" class="bg-default" ng-controller="PageController as ctrl">
+<nav-bar ng-if="!ctrl.sideNav" fluid="ctrl.sideNav"></nav-bar>
+<div class="column-layout">
+	<div class="column-layout-left bg-primary" ng-class="{open: ctrl.open}" ng-if="ctrl.sideNav">
+		<div class="column-layout-top">
+			<side-nav></side-nav>
+		</div>
+	</div>
+	<div class="column-layout-main">
+		<simple-nav
+			ng-if="ctrl.sideNav"
+			title="Settings"
+			on-burger-click="ctrl.burgerToggle()"
+			show-search="ctrl.showSearch"></simple-nav>
+
+		<!--
+		<div class="nav-toolbar" data-spy="affix" data-offset-top="66" ng-class="{
+				'affix-cancel-xs': ctrl.showSearch,
+				'affix-cancel-sm': ctrl.showSearch
+			}">
+			<div class="container m-l-0">
+				<activity-filters show-search="ctrl.showSearch"></activity-filters>
+			</div>
+		</div>
+	-->
```

Comment:
Can we get rid of commented code? Not the most important thing but lets keep it clear :D